### PR TITLE
Query multiple classifications

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds the ability to query multiple classifications.

### CODE REVIEW

File: skyportal/handlers/api/classification.py [hunk 1]
```diff
@@ -1,37 +1,138 @@
...
+            classification = Classification.get_if_accessible_by(
+                classification_id, self.current_user, raise_if_none=True
+            )
+            return self.success(data=classification)
+
+        # get owned
+        classifications = Classification.query_records_accessible_by(self.current_user)
+
+        if start_date:
+            start_date = str(arrow.get(start_date.strip()).datetime)
```
Comment (guynir42) [inline skyportal/handlers/api/classification.py:114]: Do we want to filter using a datetime object or a string?

File: skyportal/handlers/api/classification.py [hunk 2]
```diff
@@ -1,37 +1,138 @@
...
+            - classifications
+          parameters:
+          - in: query
+            name: startDate
+            nullable: true
+            schema:
+              type: string
+            description: |
+              Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
+              last_detected_at >= startDate
```
Comment (guynir42) [inline skyportal/handlers/api/classification.py:47]:
```suggestion
              created_at >= startDate
```

File: skyportal/handlers/api/classification.py [hunk 3]
```diff
@@ -1,37 +1,138 @@
...
+          - in: query
+            name: endDate
+            nullable: true
+            schema:
+              type: string
+            description: |
+              Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
+              last_detected_at <= endDate
```
Comment (guynir42) [inline skyportal/handlers/api/classification.py:55]:
```suggestion
              created_at <= endDate
```

File: skyportal/handlers/api/classification.py [hunk 4]
```diff
@@ -1,37 +1,138 @@
...
+                Classification.created_at >= start_date
+            )
+        if end_date:
+            end_date = str(arrow.get(end_date.strip()).datetime)
+            classifications = classifications.filter(
+                Classification.created_at <= end_date
+            )
+
+        total_matches = classifications.count()
+        if n_per_page is not None:
```
Comment (guynir42) [inline skyportal/handlers/api/classification.py:125]: cannot be None

**Review by guynir42 — APPROVED** [review]
Minor comments.

File: skyportal/handlers/api/classification.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/classification.py:114]: This is fine. Turns out either works.

File: skyportal/handlers/api/classification.py (hunk 4 above)
Comment (mcoughlin) [reply skyportal/handlers/api/classification.py]: Removed.

**Pull request merged**